Manufacturers.md

```diff
@@ -108,3 +108,4 @@ This is the official list of manufacturer ids (`manufacturer_id` in the target c
 |WIZZ|WIZZ|https://wizz.store/|
 |YUPF|YuPiFc|https://www.yupifc.com/|
 |ZEEZ|Zeez RC|https://www.zeezrc.com/|
+|JLDZ|YYRC||
```

Comment:
YYRC should be first.

Reply:
I have placed it in the first place. If there are any issues, please let me know and I will update it;

Reply:
i've merged master and committed proper priority/location.

Reply:
still needs URL.